include/libssh2.h

```diff
@@ -145,6 +145,84 @@ typedef int libssh2_socket_t;
 #define LIBSSH2_INVALID_SOCKET -1
 #endif /* WIN32 */
 
+/*
```

Comment:
This is brutally lifted from https://github.com/bagder/curl/blob/master/lib/config-win32.h#L631.  I'd love for it to live in a win32-specific place, but since this is public and doesn't depend on anything at configure-time this is what I went with.